.github/workflows/e2e-tests.yml

```diff
@@ -101,7 +101,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.28.1]
+        kubernetes: [v1.29.0, v1.28.1]
```

Comment:
It should be enough to run a single version here, the other jobs have the complete matrix.
Also, I think we added this when we couldn't deploy ~1.28 on bare metal and this was introduced as a workaround. If this is the case, maybe we should just remove this job altogether.

Reply:
Yeah, I can see `v1.28.1` in baremetal workflow too. It should be okay to delete this.